README.md

```diff
@@ -0,0 +1,5 @@
+# Webots Animation Template
+
+[![Webots Badge](https://badgen.net/badge/icon/Preview%20simulation?label=Webots)](https://cyberbotics.github.io/webots-animation-template/)
+
+This example shows how to generate Webots animation previews in CI using [Webots Animation](https://github.com/marketplace/actions/webots-animation) GitHub Action.
```

Comment:
I would add explicit links to the online animations as I am not sure everybody will understand they should click the badge to see the simulation playback.